src/transaction.js

```diff
@@ -408,6 +405,35 @@ const transaction = {
     return weight + 1e-6;
   },
 
+  /**
+   * Calculate the sum of outputs. Authority outputs are ignored.
+   * 
+   * @param {Array} outputs
+   * [{
+   *  'address': str,
+   *  'value': int,
+   *  'tokenData': int,
+   * }]
+   *
+   * @return {number} Sum of outputs
+   * @memberof Transaction
+   * @inner
+   */
+  getOutputsSum(outputs) {
+    let sumOutputs = 0;
+    for (let output of outputs) {
+      // XXX wallet.js has method isAuthorityOutput, but it expects an output
```

Comment:
So maybe we should create a method `isTokenDataAuthority` that receives the token data and has only the line `return output.tokenData & TOKEN_AUTHORITY_MASK`.  This method would be used by both methods `isAuthorityOutput` and `getOutputsSum `

Reply:
Done